hack/deployer/config/plans.yml

```diff
@@ -75,11 +75,12 @@ plans:
   clusterName: dev
   provider: ocp
   machineType: n1-standard-8
-  serviceAccount: false
+  serviceAccount: true
   ocp:
     region: europe-west1
     nodeCount: 3
-    overwriteDefaultKubeconfig: false
+    overwriteDefaultKubeconfig: true
+    useNonDefaultCloudSdkPath: true
```

Comment:
I think those are the defaults we'll want when deploying ocp clusters for dev most of the times.